src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Ethereum security and scams
+description: Staying safe on Ethereum
+lang: en
+sidebar: true
+---
+
+## Staying Safe on Ethereum
+
+With interest in cryptocurrencies growing exponentially, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.
```

Comment:
```suggestion
With interest in cryptocurrencies growing, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.
```
Not sure we should say exponential growth - at some point this might not be the case.

Reply:
> at some point this might not be the case.

Hopefully it does but agreed 🤪